src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -33,15 +33,15 @@ public DeleteCommand(Index targetIndex) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Person> lastShownList = model.getFilteredPersonList();
+        List<Client> lastShownList = model.getFilteredClientList();
 
         if (targetIndex.getZeroBased() >= lastShownList.size()) {
-            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+            throw new CommandException(Messages.MESSAGE_INVALID_CLIENT_DISPLAYED_INDEX);
         }
 
-        Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
-        model.deletePerson(personToDelete);
-        return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, personToDelete));
+        Client personToDelete = lastShownList.get(targetIndex.getZeroBased());
+        model.deleteClient(personToDelete);
```

Comment:
model.deleteClient(clientToDelete)

Reply:
As with the follow up requests, references to person have been changed.